packages/hawtio/src/plugins/connect/jolokia-service.ts

```diff
@@ -547,3 +567,8 @@ class DummyJolokia implements IJolokia {
 /* eslint-enable @typescript-eslint/no-unused-vars */
 
 export const jolokiaService = new JolokiaService()
+
+export const exportedOnlyForTesting = {
+  JolokiaService,
+  DummyJolokia,
+}
```

Comment:
I use this for the same purpose. It's better to follow the same naming.
https://github.com/hawtio/hawtio-next/blob/main/packages/hawtio/src/core/config-manager.ts#L149-L152
```
// Export non-exported definitions for testing
export const __testing__ = {
  ConfigManager,
}
```